### PR TITLE
Add posibility to get market maker address from URL

### PR DESCRIPTION
**Pull request by dasanra**
Refactor app loading to call web3 load only once

### CODE REVIEW

File: app/src/MarketTable/CategoricalMarketTable/index.js [hunk 1]
```diff
@@ -29,10 +29,11 @@ const MarketTable = ({
 }) => {
   useEffect(() => {
     resetMarketSelections();
+    console.log("resetMarketSelections categorical");
```
Comment (andre-meyer) [inline app/src/MarketTable/CategoricalMarketTable/index.js]: Remove please? :)

File: app/src/MarketTable/ScalarMarketTable/index.js [hunk 2]
```diff
@@ -37,10 +40,11 @@ const MarketTable = ({
 }) => {
   useEffect(() => {
     resetMarketSelections();
+    console.log("resetMarketSelections scalar");
```
Comment (andre-meyer) [inline app/src/MarketTable/ScalarMarketTable/index.js]: Remove please

File: app/src/MarketTable/CategoricalMarketTable/index.js (hunk 1 above)
Comment (dasanra) [reply app/src/MarketTable/CategoricalMarketTable/index.js]: Yes, that's the WIP part. I'm still having isssues reseting this variable after marketMakerAddress change

File: app/src/MarketTable/ScalarMarketTable/index.js (hunk 2 above)
Comment (dasanra) [reply app/src/MarketTable/ScalarMarketTable/index.js]: Done

**Pull request merged**